.gitignore

```diff
@@ -10,7 +10,7 @@ Thumbs.db
 /Apps/CesiumViewer/Gallery/gallery-index.js
 
 /Apps/Sandcastle/jsHintOptions.js
-/Apps/Sandcastle/gallery/gallery-index.js
+#/Apps/Sandcastle/gallery/gallery-index.js
```

Comment:
Can you revert this change?

Reply:
We don't want to commit `gallery-index.js`.  It is generated automatically when you run `npm run build`